recipes/protobuf/all/conanfile.py

```diff
@@ -48,20 +42,23 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-            if self.settings.os == "Windows" and self.settings.compiler in ["Visual Studio", "clang"] and "MT" in self.settings.compiler.runtime:
+            if self.settings.compiler.get_safe("runtime") in ["MT", "MTd"]:
```

Comment:
Not sure if we already care about this, but this seems not to be future proof with the settings for the new MSVC compiler

https://github.com/conan-io/conan/blob/03b3d71890c537df1154b3d04cc35ed2654a5894/conans/client/conf/__init__.py#L87

Reply:
Not a regression in any case, as the old code did not support the MSVC compiler settings either

Reply:
I can add `"static"`. An new method in tools would be welcome to avoid fragile verifications. There are many recipes like that.